app/views/base/layout.scala.html

```diff
@@ -195,21 +195,21 @@
     <div id="footer_wrap">
       <div class="footer">
         <div class="right">
-          <a href="http://github.com/ornicar/lila" target="_blank">Source Code</a> |
-          <a href="@routes.Main.developers">Developers</a><br />
-          <a href="@routes.Page.contact">Contact</a> |
-          <a href="@routes.Page.thanks">Thanks</a> |
-          <a href="@routes.Page.tos">ToS</a>
+          <a href="http://github.com/ornicar/lila" target="_blank">@trans.sourceCode()</a> |
+          <a href="@routes.Main.developers">@trans.developers()</a><br />
+          <a href="@routes.Page.contact">@trans.contact()</a> |
+          <a href="@routes.Page.thanks">@trans.thanks()</a> |
+          <a href="@routes.Page.tos">@trans.termsOfService()</a>
```

Comment:
I think you're going to far, here. All these resources require knowledge of english, anyway.